Makefile

```diff
@@ -15,12 +15,16 @@ default: deploy
 # not really sure what to do with this target yet
 deploy-client: 
 
-deploy:
+deploy: deploy-service-scripts deploy-lib
+
+deploy-lib:
 	rm -rf $(SERVICE_DIR)/venv
 	virtualenv $(SERVICE_DIR)/venv
+	. $(SERVICE_DIR)/venv/bin/activate && pip install .
+
+deploy-service-scripts:
 	cp $(SCRIPTS_DIR)/start_service $(SCRIPTS_DIR)/stop_service $(SERVICE_DIR)/
 	rsync -avzP $(LBIN_DIR) $(SERVICE_DIR)/
```

Comment:
Ah, cool!  So don't merge yet, I'll fix the Makefile and the start script to use the venv version.